tex/calculus/taylor.tex

```diff
@@ -199,8 +199,8 @@ \section{Differentiating them}
 	[A description of power series coefficients]
 	Let $\sum_{n \ge 0} a_n z^n$ be a power series
 	with radius of convergence $R > 0$,
-	and consider the corresponding function $f(x)$ as above
-	Then
+	and consider the corresponding function $f(x)$ as above,
```

Comment:
![image](https://user-images.githubusercontent.com/38930062/69907729-3e9dc700-138f-11ea-8149-6c736f5f8cb8.png)

This was displayed like so. Another way to fix this would have been a period, but I went with a comma. Happy to change it.

Reply:
I went with the period. Thanks!